### PR TITLE
Create zh_cn.json

### PR DESCRIPTION
**Pull request by qsefthuopq**

### CODE REVIEW

File: src/main/resources/assets/megane/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "config.waila.megane"                    : "眼镜",
```
Comment (deirn) [inline src/main/resources/assets/megane/lang/zh_cn.json:2]: cool, but i dont think you need to translate the mod name. it just weird, you know :P

Comment (qsefthuopq) [reply src/main/resources/assets/megane/lang/zh_cn.json:2]: Alright,I know it's Japanese.

Comment (deirn) [reply src/main/resources/assets/megane/lang/zh_cn.json:2]: yeah, but keep it as "megane" please

Comment (deirn) [reply src/main/resources/assets/megane/lang/zh_cn.json:2]: it's there because hwyla forced to use lang key

**Comment by qsefthuopq** [comment]
Under MIT license,I'll introduce this mod in MCBBS and Minecraft Community Wiki for Chinese introduction later.

**Comment by deirn** [comment]
sure, do whatever you want with it :+1:

**Pull request merged**